tasks/node-setup.yml

```diff
@@ -1,4 +1,31 @@
 ---
+
+- block:
+
+  - name: Check if bridge-nf-call-iptables key exists
+    command: "sysctl net.bridge.bridge-nf-call-iptables"
+    failed_when: false
+    changed_when: false
+    register: sysctl_bridge_nf_call_iptables
+
+  - name: Enable bridge-nf-call tables
+    sysctl:
+      name: "{{ item }}"
+      state: present
+      sysctl_file: "{{ sysctl_file_path }}"
+      value: 1
+      reload: yes
+    when: sysctl_bridge_nf_call_iptables.rc == 0
```

Comment:
You only set the values to 1 if the key does not exist. But what if key exists and is set to 0?

Reply:
That's not what is happening here.  The _return code_ of the `sysctl_bridge_nf_call_iptables` variable is being checked, not the value of the key.  The return code will be 0 if the command "sysctl net.bridge.bridge-nf-call-iptables" returns anything at all (suggesting that the sysctl key net.bridge.bridge-nf-call-iptables exists on the system), regardless of its value.

Reply:
Additionally, if the key exists and it is set to 0, this task will set it to 1.   If its already set to 1, it wont do anything